File: src/codes/Users_Manual/transducer_0.cpp
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~auto autd =
//~autd3::ControllerBuilder({autd3::AUTD3(autd3::Vector3::Zero())}).open(autd3::link::Nop::builder());
const auto tr = autd.geometry()[0][0];
const auto idx = tr.idx();
const auto position = tr.position();
//~return 0; }

File: src/codes/Users_Manual/transducer_0.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
~using var autd = Controller.Builder([new AUTD3(Vector3.Zero)]).Open(Nop.Builder());
var tr = autd.Geometry[0][0];
var trIdx = tr.Idx;
var position = tr.Position;